okdata/cli/commands/datasets/wizards.py

```diff
@@ -56,6 +57,17 @@ def start(self):
         env = self.command.opt("env")
         choices = run_questionnaire(*qs_create())
 
+        confirm_to_continue(
+            "Will create a new dataset '{}'.{}".format(
+                choices["title"],
+                (
+                    "\n\nData uploaded to the dataset will be PUBLICLY AVAILABLE ON THE INTERNET.\n"
```

Comment:
Blir kanskje for mye å fargelegge litt her?

Reply:
Det skader vel ikke! Men kanskje det da kan gjøres i en egen PR hvor andre lignende advarsler også fargeleges. Kanskje fet skrift kunne holdt når jeg tenker meg om. :stuck_out_tongue: